test/unit/foreman/access_permissions_test.rb

```diff
@@ -10,59 +11,8 @@
 #
 # In particular, it's important that actions for AJAX requests are added to
 # an appropriate permission so views using those requests function.
-class AccessPermissionsTest < ActiveSupport::TestCase
-  MAY_SKIP_REQUIRE_LOGIN = [
```

Comment:
> If i skip the test completely (and don't run my own with my added routes), i'm not checking that my added routes within the foreman routes are fully 'permissioned'.

Rather than skipping the _complete_ test, only skip the individual tests for routes that you want to exclude (for whatever legitimate reason). This way the AccessPermissionTest will always pass. Needing to redefine the test to add skips/exclusions means the _original_ test has to be disabled to prevent failures, which shouldn't be necessary.